portal/src/main/webapp/WEB-INF/jsp/oncoprint/main.jsp

```diff
@@ -107,6 +116,17 @@
                            <div class="checkbox"><label><input type="checkbox" name="cbioportal" value="cbioportal"/> cBioPortal  >= <input type="text" id="cbioportal_threshold" style="width:35px;"/></label></div>
                            <div class="checkbox"><label><input type="checkbox" name="cosmic" value="cosmic"> COSMIC  >= <input type="text" id="cosmic_threshold" style="width:35px;"/></label></div>
                            <div class="checkbox"><label><input type="checkbox" name="hide_unknown" value="hide_unknown"> Hide putative passenger mutations</label></div>
+                           <% if (!GlobalProperties.getBinaryCustomAnnotationMenuLabel().trim().equals("")) { %>
+                           <div class="checkbox"><label><input type="checkbox" name="binary_filter" value="binary_filter"/> <c:out value="${GlobalProperties.getBinaryCustomAnnotationMenuLabel()}" /> <img id="colorby_binary_filter_info" src="images/button-yellow.png" alt="binary filter" style="height:15px; width:15px; cursor:pointer;"/></label></div>
+                           <% }
+                           if (!GlobalProperties.getMultiCustomAnnotationMenuLabel().trim().equals("")) { %>
+                           <div class="checkbox"><label><input type="checkbox" name="multi_filter" value="multi_filter"/> <c:out value="${GlobalProperties.getMultiCustomAnnotationMenuLabel()}" /> <img id="colorby_multi_filter_info" src="images/button-seagreen.png" alt="multi filter" style="height:15px; width:15px; cursor:pointer;"/></label></div>
+                           <div id="multi_values" style="margin-left: 30px;">
+                               <c:forEach var="multiValue" items="${DaoMutation.getMultiFilterValues(requestScope[QueryBuilder.CANCER_STUDY_ID])}" varStatus="loop">
+                                   <div class="checkbox"><label><input type="checkbox" name="multi_filter_${loop.index}" value="${fn:escapeXml(multiValue)}" /> <c:out value="${multiValue}" /></label></div>
+                               </c:forEach>
+                           </div>
+                           <% } %>
```

Comment:
Change colors of cbioportal_driver_filter and cbioportal_multi_filter, the greens icons are confusing with greens in legend below OncoPrint.